CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-08-21
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+  * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+* Kubernetes 1.21 version has been deprecated as of July 31st, 2022.
+* Some AKS labels have been deprecated with the Kubernetes 1.24 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* The Open Service Mesh addon has been updated from version 1.1.1 to version [1.2.0](https://github.com/openservicemesh/osm/releases/tag/v1.2.0) for AKS clusters running 1.24.0+. Please note the breaking changes mentioned in the [version 1.2.0 release notes](https://github.com/openservicemesh/osm/releases/tag/v1.2.0)
+
+### Release notes
+
+* Bug fixes
+  * v20220724 contains a bug that will cause clusters running the Open Service Mesh addon to fail when upgrading to v1.24.0, and therefore the upgrade will fail as well. This bug was hotfixed in the v20220731 release. It was also fixed in the v20220807 release but the hotfix needs to rollout to customers already seeing the bug in the mean time. 
```

Comment:
so which release was the fix actually in? 2022-08-07? if yes, we should move this point there and update the release description to capture the same. on rollout of this specifically, may be we call it out under announcement section? (as the fix itself was from a previous release and not in this one)

Reply:
@allyford what is the date for SOv cloud and the Airgap as we mention usually at the very top ?

Reply:
On the deprecation of K8s 1.21 version should we add any action for customers as well ?

Reply:
> so which release was the fix actually in? 2022-08-07? if yes, we should move this point there and update the release description to capture the same. on rollout of this specifically, may be we call it out under announcement section? (as the fix itself was from a previous release and not in this one)

Update from engineer - the bug was fixed and hotfixed so there are no versions currently with the bug. Removing as per recommendation

Reply:
> @allyford what is the date for SOv cloud and the Airgap as we mention usually at the very top ?

As of the previous release notes, we are referencing the AKS Release tracker instead

Reply:
> On the deprecation of K8s 1.21 version should we add any action for customers as well ?

Adding documentation link for upgrade